lib/modules/dosomething/dosomething_user/dosomething_user.badge.inc

```diff
@@ -33,6 +33,11 @@ function dosomething_user_get_bages($uid) {
   }
 
   // Get special type badges.
+  // $special_badge_campaigns = [nids of whatever ones we want here]
```

Comment:
@katiecrane I'd say let's use `362` for now/always to demo with because that NID is the same on prod/staging/dev/everywhere AND we can do a big ol' pile badge